<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::ContentBlockingDialog</class>
 <widget class="QDialog" name="Otter::ContentBlockingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Content Blocking</string>
  </property>
  <layout class="QVBoxLayout" name="contentBlockingVerticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="profilesVerticalLayout">
       <item>
        <widget class="QLabel" name="profilesHeaderLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Profiles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="profilesDescriptionLabel">
         <property name="text">
          <string>Select lists which you want to use for content blocking (AdBlock Plus compatible):</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="profilesHorizontalLayout">
         <item>
          <widget class="Otter::ItemViewWidget" name="profilesViewWidget">
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustIgnored</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="autoExpandDelay">
            <number>-1</number>
           </property>
           <attribute name="headerStretchLastSection">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="profilesButtonsLayout" stretch="0,0,0,0,1">
           <item>
            <widget class="QPushButton" name="addProfileButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editProfileButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Edit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="updateProfileButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Update</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeProfileButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="profilesVerticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="settingsHeaderLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="enableCosmeticFiltersCheckBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Enable cosmetic filters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="enableWildcardsCheckBox">
         <property name="text">
          <string>Enable wildcard expressions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="enableCustomRulesCheckBox">
         <property name="text">
          <string>Enable custom rules</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="customRulesTab">
      <attribute name="title">
       <string>Custom Rules</string>
      </attribute>
      <layout class="QVBoxLayout" name="customVerticalLayout">
       <item>
        <layout class="QHBoxLayout" name="customRulesHorizontalLayout">
         <item>
          <widget class="Otter::ItemViewWidget" name="customRulesViewWidget">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="customRulesButtonsLayout" stretch="0,0,0,1">
           <item>
            <widget class="QPushButton" name="addRuleButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editRuleButton">
             <property name="text">
              <string>Edit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeRuleButton">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="customRulesVerticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="confirmButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::ItemViewWidget</class>
   <extends>QTreeView</extends>
   <header>src/ui/ItemViewWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
